src/components/posts/State.tsx

```diff
@@ -0,0 +1,15 @@
+import { atom, atomFamily } from 'recoil'
+import { IComment } from './comments/CommentList'
+import { IPost } from './PostList'
+
+const postAtom = atom<IPost[]>({
```

Comment:
Since it holds multiple comments and not a just a single one, name of the atom should also be plural to reflect that.